src/resources/globals.scss

```diff
@@ -0,0 +1,35 @@
+// Fonts
+$font-400: 14px;
+$font-300: 12px;
+
+// Spacing
+$space-940: 360px;
```

Comment:
I know we're using this naming in Kenshoo internally but it doesn't make much sense in my opinion. Sins of the fathers... :(

$font-default-size: 12px;
$font-large-size: 14px;
makes more sense.

The spacings, lets align the param name with the actual value etc.

Reply:
ok

Reply:
done